tests/plugins/richcombo/richcombo.js

```diff
@@ -89,6 +93,62 @@ bender.test( {
 		} );
 	},
 
+	// (#4493)
+	'test richcombo aria-labelledby attribute points to label element of the combobox': function() {
+		var editor = this.editor,
+			combo = editor.ui.get( 'custom_combo' ),
+			comboBtnSelector = '#cke_' + combo.id,
+			comboBtn = CKEDITOR.document.findOne( comboBtnSelector + ' .cke_combo_button' ),
+			comboBtnLabelElementId = comboBtnSelector.substr( 1 ) + '_label';
+
+		combo.createPanel( editor );
+
+		assert.areEqual( comboBtnLabelElementId, comboBtn.getAttribute( 'aria-labelledby' ), 'Aria-labelledby attribute should point to element with the combo label.' );
+	},
+
+	// (#4493)
+	'test richcombo initial label': function() {
+		var editor = this.editor,
+			combo = editor.ui.get( 'custom_combo' ),
+			comboBtnSelector = '#cke_' + combo.id,
+			comboBtnLabelSelector = comboBtnSelector + '_label',
+			comboBtnLabel = CKEDITOR.document.findOne( comboBtnLabelSelector );
+
+		combo.createPanel( editor );
+
+		assert.areEqual( initialLabel, comboBtnLabel.getHtml() );
+	},
+
+	// (#4493)
+	'test richcombo label after selecting some option': function() {
+		var editor = this.editor,
+			combo = editor.ui.get( 'custom_combo_with_options' ),
+			comboBtnSelector = '#cke_' + combo.id,
+			comboBtnLabelSelector = comboBtnSelector + '_label',
+			comboBtnLabel = CKEDITOR.document.findOne( comboBtnLabelSelector ),
+			expectedLabel = 'ONE, ' + initialLabel;
+
+		combo.createPanel( editor );
+		combo.setValue( 'one', 'ONE' );
+
+		assert.areEqual( expectedLabel, comboBtnLabel.getHtml() );
+	},
+
+	// (#4493)
+	'test richcombo label after returning to defaul value': function() {
```

Comment:
I would improve this test case name in the same way as the previous one. This one could be something like `test richcombo label contains default label after richcombo returs to default value`